test/integration/api/services_controller_test.rb

```diff
@@ -210,4 +210,19 @@ def setup
       assert_equal 1, Service.last.backend_api_configs.count
     end
   end
+
+  class ServiceCreateTest < Api::ServicesControllerTest
+    test 'create error shows the right flash message' do
+      post admin_services_path, service: { name: '' }
+      assert_equal 'Couldn\'t create Product. Check your Plan limits', flash[:error]
+
+      @provider.settings.allow_multiple_services!
+
+      post admin_services_path, service: { name: '' }
+      assert_equal 'Name can\'t be blank', flash[:error]
+
+      post admin_services_path, service: { name: 'example-service', system_name: '###' }
+      assert_equal 'System name invalid. Only ASCII letters, numbers, dashes and underscores are allowed.', flash[:error]
```

Comment:
@didierofrivia I also respected your cucumber for you to see that it passes: https://github.com/3scale/porta/pull/1398/files#diff-9186c909b21b5e5da01c4f9ab0d72b89R40

But I admit that I felt tempted to remove it because our cukes 🥒 usually become unmaintainable nightmares 😄